server/apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -175,115 +128,3 @@ def format(self, article, subscriber):
 
     def can_format(self, format_type, article):
         return format_type == 'AAP IPNEWS' and article['type'] in ['text', 'preformatted']
```

Comment:
@akintolga I have created this content type named tuple. I think @sivakuna-aap referring to that. I will do that refactor this sprint.